### DISCUSSION TITLE
Nova APIs

### DISCUSSION THREAD

**Discussion by christinezaki** [laravel/nova-issues#3458] [Q&A] [answered]
- Laravel Version: 7
- Nova Version: 3
- PHP Version: 7.4
- Database Driver & Version: mysql 
- Operating System and Version: windows 10

I am developing a mobile app and i want to directly call nova APIs is there a way to do this?
when i tried to call {{SERVER_IP}}/api/users?editing=true&editMode=create via postman the response was 419

**Comment by crynobone** [comment, marked as answer]
> Do note that Nova API is designed to work directly just for Laravel Nova and it doesn't belong in Public API. Laravel Team may require the make breaking changes to it in patch releases. Please use it with caution.

**Reply by christinezaki** [reply]
could you please help how can i fix 419 response code
